### PR TITLE
[onert] Add shape inference for Cos

### PR DESCRIPTION
**Pull request by intom**
This adds shape inference for Cos operation.

I refer to this PR https://github.com/Samsung/ONE/pull/1380

Signed-off-by: sungho22.lee <sungho22.lee@samsung.com>

### CODE REVIEW

File: runtime/onert/core/src/util/shapeinf/Cos.cc [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+namespace onert
+{
+namespace shape_inference
+{
+
+void StaticInferer::visit(const ir::operation::Cos &op)
+{
+  const auto input_idx{op.getInputs().at(ir::operation::Cos::Input::INPUT)};
+  const auto &input = _operands.at(input_idx);
```
Comment (YiHyunJin) [inline runtime/onert/core/src/util/shapeinf/Cos.cc:27]: Could you change it to handleSimpleUnaryOp?
https://github.com/Samsung/ONE/pull/1380

Comment (intom) [reply runtime/onert/core/src/util/shapeinf/Cos.cc]: > Could you change it to handleSimpleUnaryOp?
> #1380

I changed

**Comment by intom** [comment]
@nnfw-bot test x64-builder

**Comment by intom** [comment]
@nnfw-bot rerun

File: tests/nnapi/specs/Ex/cos_ex_dynamic_nnfw.mod.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+Testing Cos op when the input is dynamic.
+      input [1, 2, 3]  shape [3]  (value of shape will be [1, 2, 3])
+          |             |
+          +-------------+
+          |
+       Reshape (added by DynamicInputGenerator since it generates its output to be dynamic)
+          |
+          | dynamic tensor at compilation time but the shape will be [1, 2, 3] at execution time
+          |
+         Abs
```
Comment (hyunsik-yoon) [inline tests/nnapi/specs/Ex/cos_ex_dynamic_nnfw.mod.py:33]: maybe `Cos`?

Comment (intom) [reply tests/nnapi/specs/Ex/cos_ex_dynamic_nnfw.mod.py]: @hyunsik-yoon I modified. Is there anything to add?

**Comment by glistening** [comment]
@intom Please update commit message, which has no word except for Signed-off.

**Pull request merged**